docs/content/documentation/getting-started/installation.md

```diff
@@ -155,6 +155,12 @@ From a terminal, you can now run the following command:
 $ cargo build --release
 ```
 
+If you encountered compilation errors like `error: failed to run custom build command for 'ring v0.16.20'`, you can try the command below instead:
+
+```sh
+$ cargo build --release --no-default-features --features=native-tls
```

Comment:
can we add that command to CI to make sure it runs well? Without the `--release` argument of course

Reply:
> can we add that command to CI to make sure it runs well? Without the `--release` argument of course

Thank you for your reply. I will add a new CI job to test this case.

Reply:
I think just a step is fine, no need for a new job

Reply:
Done. And all the CI tests seem to be passing.